src/components/ArticleList.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+
+
+const ArticleList = () => {
+    const [articles, setArticles] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
This should really start out as false and then the we set it to be true in the useEffect.